Python/ibmcloudsql/cos.py

```diff
@@ -449,26 +499,38 @@ def list_cos_objects(self, cos_url):
             raise ValueError(msg)
         self.logon()
 
+        if cos_url[-1] != "/":
```

Comment:
@thoangtrvn I did a system test with ibmcloudsql/test.py and discovered a regression in rename_exact_result() function, which is caused by object listing foe the result objects of a job now not returning the zero by folder object itself anymore. But rename_exact_result() is based on a certain pattern of objects being produced to decide whether it is a standard query result that can be renamed to single object or not. This is caused by this change here. Why are you appending a `/` now always? Can we undo this again?

Reply:
undo it.